.github/workflows/run-tox.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
Why this change? Could it be that this version does not have py38 => tox does not run it?

Reply:
That's what I wanted to check but no, tox still does do anything...

Reply:
Ah I found it: py38 was missing in tox.ini.

Reply:
good